source/app.d

```diff
@@ -72,11 +73,12 @@ class ImageFitness
     {
         SuperImage source = image(destination.width, destination.height);
         Circle[] shapes;
-        foreach(index; 0 .. (genom.length / populationSize))
+        size_t numberOfShapesInPopulation = genom.length / populationSize;
+        shapes.reserve(numberOfShapesInPopulation);
```

Comment:
Tohle je stejně zcela zbytečná alokace - alokuje se to jenom proto, aby se to pak v rasterize prošlo.